defs.bzl

```diff
@@ -22,6 +24,7 @@ _gazelle_cabal_dependencies = repository_rule(
     local = True,
     attrs = {
         "aeson": attr.label(default = "@stackage//:aeson"),
+        "exceptions": attr.label(default = "@stackage//:exceptions"),
```

Comment:
Adding dependencies could require the user to be aware of them. It would be better to avoid it if it isn't essential.

Reply:
I agree, I'll drop this.

Reply:
Done